docs/integrations/ml-dist.rst

```diff
@@ -0,0 +1,512 @@
+
+- `A running CrateDB cluster`_
+- `A AWS S3 storage bucket`_
+- A AWS config file at ``~/.aws/config`` containing:
+
+.. code-block:: python
+
+    [default]
+    aws_access_key_id=<key>
+    aws_secret_access_key=<access_key>
+
+- `Python`_
+    - The `tensorflow`_ library
+    - The `numpy`_ library
+    - The `sklearn`_ library
+    - The `pandas`_ library
+    - The `psycopg2`_ library
+    - The `boto3`_ library
+    - The `joblib`_ library
+    - The `matplotlib`_ library
+
+
+CrateDB
+-------
+
+In this first step we need to make sure that we create the table that holds the pump_sensor
+data set as well as the table that will be saving the predictions:
+
+.. code-block:: sql
+
+    CREATE TABLE "ml"."pump_sensors"
+    (
+       "id" INTEGER,
+       "timestamp" TEXT,
+       "sensor_00" REAL,
+       "sensor_01" REAL,
+       "sensor_02" REAL,
+       "sensor_03" REAL,
+       "sensor_04" REAL,
+       "sensor_05" REAL,
+       "sensor_06" REAL,
+       "sensor_07" REAL,
+       "sensor_08" REAL,
+       "sensor_09" REAL,
+       "sensor_10" REAL,
+       "sensor_11" REAL,
+       "sensor_12" REAL,
+       "sensor_13" REAL,
+       "sensor_14" REAL,
+       "sensor_15" REAL,
+       "sensor_16" REAL,
+       "sensor_17" REAL,
+       "sensor_18" REAL,
+       "sensor_19" REAL,
+       "sensor_20" REAL,
+       "sensor_21" REAL,
+       "sensor_22" REAL,
+       "sensor_23" REAL,
+       "sensor_24" REAL,
+       "sensor_25" REAL,
+       "sensor_26" REAL,
+       "sensor_27" REAL,
+       "sensor_28" REAL,
+       "sensor_29" REAL,
+       "sensor_30" REAL,
+       "sensor_31" REAL,
+       "sensor_32" REAL,
+       "sensor_33" REAL,
+       "sensor_34" REAL,
+       "sensor_35" REAL,
+       "sensor_36" REAL,
+       "sensor_37" REAL,
+       "sensor_38" REAL,
+       "sensor_39" REAL,
+       "sensor_40" REAL,
+       "sensor_41" REAL,
+       "sensor_42" REAL,
+       "sensor_43" REAL,
+       "sensor_44" REAL,
+       "sensor_45" REAL,
+       "sensor_46" REAL,
+       "sensor_47" REAL,
+       "sensor_48" REAL,
+       "sensor_49" REAL,
+       "sensor_50" REAL,
+       "sensor_51" REAL,
+       "machine_status" TEXT
+    );
+
+    CREATE TABLE "ml"."pump_sensors_predictions"
+    (
+       "prediction" REAL,
+       "ts" TIMESTAMP WITH TIME ZONE,
+       "model_name" TEXT
+    );
+
+After creating the tables we will start with looking at our data before we will
+insert data to our table, since the data might not be in a state where it can
+be easily inserted into our CrateDB instance.
+
+Assumption
+----------
+
+The python code is considered to all be in a single ``__main__`` function
+if nothing else was explicitly stated. Imports should all be at top of the
+file, while the code snippets follow in order of each other from the top
+to the bottom of the file.
+
+Examining The Data
+------------------
+
+We will begin with loading in our data to a pandas dataframe, so we can have
+a look at it and make adjustments accordingly.
```

Comment:
I think this data examination/cleansing section is probably not necessary. We could maybe say in the prerequisites that we have some sample data that has been cleaned up for processing, since this section doesnt really involve doing anything with CrateDB.